src/psycop/common/feature_generation/application_modules/project_setup.py

```diff
@@ -96,7 +88,7 @@ def get_project_info(
 
     return ProjectInfo(
         project_path=proj_path,
-        feature_set_path=feature_set_path,
+        flattened_dataset_path=feature_set_path,
```

Comment:
Refactor `get_project_info` and `create_feature_set_path` - where are they used? 

If somewhere important, ensure covered by a test. If nowhere, delete.